### PR TITLE
AT-90 changed to accept response status

### PR DESCRIPTION
**Pull request by grantwhiteman**
Frontend changed to accept new API response.
Now shows success/failure then a list of failures.

https://dfacademy.atlassian.net/jira/software/projects/AT/boards/4?issueParent=10138&selectedIssue=AT-90

### CODE REVIEW

**Review by benclayton98 — APPROVED** [review]
Good work Grant, can't see any obvious errors or code smells. LGTM

File: src/components/Upload/Upload.js [hunk 1]
```diff
@@ -9,14 +9,14 @@ class Upload extends Component {
...
-			errors         : [],
+			response       : { status: "", errors : [] },
 			color          : '',
 			invalid		   : ''
 		};
 	};
 
 	setTheColor() {
-		if (this.state.errors[0] === 'Updated the database successfully.') {
+		if (this.state.response.status === 'success') {
```
Comment (dearshrewdwit) [inline src/components/Upload/Upload.js:19]: In general store a relationship like this in an obj 

const colours = {
  success: "green",
  failure: "red"
}

Which then removes the need for the `setTheColor` method

File: src/components/Upload/Upload.js [hunk 2]
```diff
@@ -52,7 +52,7 @@ class Upload extends Component {
 		})
 			.then(response => response.json())
 			.then(data => {
-				this.setState({ errors: data.response});
+				this.setState({ response: data.response });
```
Comment (dearshrewdwit) [inline src/components/Upload/Upload.js:55]: And then use it here - 

```js
this.setState({ 
  response: data.response, 
  color: colours[data.response.status]
})
```

**Comment by dearshrewdwit** [comment]
> Good work Grant, can't see any obvious errors or code smells. LGTM

I now expect these on your team's PRs @benclayton98

File: src/components/Upload/Upload.js (hunk 1 above)
Comment (grantwhiteman) [reply src/components/Upload/Upload.js]: sounds good to me, not sure what you mean by a separate module honestly, but I put it outside the class.

File: src/components/Upload/Upload.js (hunk 2 above)
Comment (grantwhiteman) [reply src/components/Upload/Upload.js]: Love it

**Review by dearshrewdwit — APPROVED** [review]

**Review by benclayton98 — APPROVED** [review]
LGTM

**Pull request merged**